src/javascript/JContent/actions/editImage/ImageEditorDialog/ImageEditorDialog.jsx

```diff
@@ -49,8 +50,9 @@ export const ImageEditorDialog = ({
                 onClose={onClose}
         >
             <DialogTitle disableTypography id="form-dialog-title" classes={{root: 'flexRow alignCenter'}}>
-                <Typography variant="title" className={styles.title}>{name}</Typography>
-                <div>
+                <Typography variant="heading">Edit {name}</Typography>
```

Comment:
"Edit" should use a translation instead of a hardcoded String

Reply:
good catch! fixed